src/store/MLResultsStore.js

```diff
@@ -0,0 +1,56 @@
+import { types } from 'mobx-state-tree'
+import { ASYNC_STATES } from '@util'
+import config from '@config'
+import superagent from 'superagent'
+
+const MLResultsStore = types.model('MLResultsStore', {
+  
+  status: types.optional(types.string, ASYNC_STATES.IDLE),
+  data: types.optional(types.frozen({}), {}),
```

Comment:
Should this just be `types.frozen({})`?

Reply:
👍 It should be, you're right. I should have paid more attention since the first `{}` already defines the default value.